### PR TITLE
[OSDEV-852] Make creating Kafka topic automatically after deploy

### PR DESCRIPTION
**Pull request by Innavin369**
* Add script that creates Kafka topic Ising aws-cli tools
* Add a steps that:
  * update Kafka-tool ECR with image
  * call script that create Kafka topic
* Add preprod to ECSMANAGE_ENVIRONMENTS

[OSDEV-852](https://opensupplyhub.atlassian.net/browse/OSDEV-852)

### CODE REVIEW

File: src/django/oar/rollbar.py [hunk 1]
```diff
@@ -16,14 +16,15 @@ def report_error_to_rollbar(
         contributor_id_msg = ''
         is_api_user_msg = ''
         if request:
-            data['user_id'] = getattr(request.user, 'id', None)
```
Comment (roman-stolar) [inline src/django/oar/rollbar.py:19]: Probably those change should be in PR.

File: src/django/api/views/facility/facility_list_view_set.py [hunk 2]
```diff
@@ -380,7 +380,9 @@ def createlist(self, request):
         try:
             rows = serializer.get_validated_rows()
         except ValidationError as err:
-            report_error_to_rollbar(request=request, file=uploaded_file)
+            report_error_to_rollbar(request=request,
```
Comment (roman-stolar) [inline src/django/api/views/facility/facility_list_view_set.py:383]: Probably those rollbar change shouldn't be in PR.

File: src/django/api/middleware.py [hunk 3]
```diff
@@ -34,12 +34,12 @@ def __call__(self, request):
                         path=request.get_full_path(),
                         response_code=response.status_code,
                     )
-        except Exception as e:
+        except Exception as err:
```
Comment (roman-stolar) [inline src/django/api/middleware.py:37]: Probably those rollbar change shouldn't be in PR.

File: src/django/api/matching.py [hunk 4]
```diff
@@ -724,11 +724,11 @@ def dedupe_record_for_match_item(item):
                             cls._gazetter.index(record)
                     cls._match_version = item['history_id']
 
-        except Exception as e:
+        except Exception as err:
             extra_data = {
                 'last_successful_facility_version': cls._facility_version,
                 'last_successful_match_version': cls._match_version}
-            report_error_to_rollbar(extra_data=extra_data, exception=e)
+            report_error_to_rollbar(extra_data=extra_data, exception=err)
```
Comment (roman-stolar) [inline src/django/api/matching.py:731]: Probably those rollbar change shouldn't be in PR.

File: src/app/src/util/util.js [hunk 5]
```diff
@@ -1097,7 +1097,9 @@ export const logErrorToRollbar = (window, error, user) => {
             const rollbarError = new Error(`${rollbarErrMsg} ${error.message}`);
             Object.assign(rollbarError, error);
 
-            window.Rollbar.error(rollbarError);
+            window.Rollbar.error(rollbarError, {
```
Comment (roman-stolar) [inline src/app/src/util/util.js:1100]: Probably those rollbar change shouldn't be in PR.

File: res.txt [hunk 6]
```diff
@@ -0,0 +1,1346 @@
+VPC ID: vpc-095a9ef7a8f0a460e
```
Comment (roman-stolar) [inline res.txt:1]: gitignore this file?

File: res.yml [hunk 7]
```diff
@@ -0,0 +1,62 @@
+VPC ID: vpc-095a9ef7a8f0a460e
```
Comment (roman-stolar) [inline res.yml:1]: gitignore this file?

File: deployment/terraform/servicediscovery-drain.sh [hunk 8]
```diff
@@ -0,0 +1,24 @@
+[ $# -ne 1 ] && echo "Usage: $0 <service-id>" && exit 1
```
Comment (roman-stolar) [inline deployment/terraform/servicediscovery-drain.sh:1]: Probably those changes exist in another PR

File: deployment/terraform/dns.tf [hunk 9]
```diff
@@ -126,4 +126,10 @@ resource "aws_service_discovery_service" "app" {
   health_check_custom_config {
     failure_threshold = 5
   }
+
+  # Remove after https://github.com/terraform-providers/terraform-provider-aws/issues/4853 is resolved
+  provisioner "local-exec" {
+    when = destroy
```
Comment (roman-stolar) [inline deployment/terraform/dns.tf:132]: Probably those changes exist in another PR

**Review by roman-stolar — APPROVED** [review]

**Review by mazursasha1990 — APPROVED** [review]

**Pull request merged**